codegen/generator/example.go

```diff
@@ -59,6 +59,13 @@ func Example(genpkg string, roots []eval.Root) ([]*codegen.File, error) {
 				files = append(files, fs...)
 			}
 		}
+		for _, f := range files {
+			if len(f.SectionTemplates) > 0 {
+				for _, s := range r.Services {
+					service.AddServiceDataMetaTypeImports(f.SectionTemplates[0], s)
```

Comment:
Please fix indentation

Reply:
What do you mean? I run gofmt on save, what do you think needs to be fixed?

Reply:
My bad! Didn't look at the actual file. Github UI confused me a bit.